docs/static/openapi.yml

```diff
@@ -33242,34 +33242,51 @@ paths:
                       type: string
                   additionalProperties: {}
       parameters:
-        - name: status
-          in: path
-          required: true
-          type: string
-          enum:
-            - scheme
-            - prototype
-            - trial
-            - permanent
-            - suspended
-            - banned
-            - bannedSoon
-            - bannedVerySoon
-            - none
-            - playable
-            - unplayable
         - name: owner
           in: query
           required: false
           type: string
-        - name: cardType
+        - name: statuses
```

Comment:
why plural? a card can only have 1 status right?

Reply:
yes, but dont u query a list of cards?

Reply:
Eh yes, but you can only specify 1 status? So what you say is like "Give me all card**s** with status "permanent". 
Example from real life:
You want to buy 4 apple**s**, the apples are plural, then the nice lady in the marketplace asks you "Which sort of apples do you want?", then you yell "BOSKOP". The lady does not ask "Which sort**s** of apples do you want?"